src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+public abstract class Figure implements FigureArea {
+    protected String color;
+
+    public Figure() {
+        ColorSupplier color = new ColorSupplier();
+        this.color = color.getRandomColor();

```

Comment:
Why are you assigning random color by default here? Use constructor to allow passing the value while creating an object.